README.md

```diff
@@ -179,3 +179,22 @@ will not be emitted by `onEvent`. For more information, see the docs.
  * `RemoteRoom` - A representation of a third-party room.
  * `MatrixUser` - A representation of a matrix user.
  * `RemoteUser` - A representation of a third-party user.
+
+
+## Signaling Bridge Errors
+This section applies when you are using `Bridge` and want to notify your users
+about problems while processing their events.
+
+One thing the bridge requires you to do is fulfilling or rejecting the
+`request` promise which is handed to you as argument of the
+`controller.onEvent` callback. When rejecting the promise the `Error` you
+reject with does matter:
+
+- On most `Error`s the bridge will simply notify the homeserver that the event
```

Comment:
I would reorder this list, and rephrase this to say "On all other Error types, the bridge will..."